Makefile.am

```diff
@@ -617,6 +618,14 @@ builtin_modules += android_wakelock
 builtin_sources += plugins/android-wakelock.c
 endif
 
+if ACCOUNTSSETTINGS
```

Comment:
Although much of the code is generic to a device running systemd, the internal DBus interface used to access the AccountService is specific to Ubuntu Touch.  As such, I would suggest prefixing the plugin name ( and possible the define ) with "ubuntu-", similar to our provisioning plugin.